lib/install/templates/assets.rb

```diff
@@ -0,0 +1,45 @@
+# Installs modules to enable assets linking, compiling and digesting
+# with webpack
+
+require "json"
+require "webpacker/package_json"
+
+# Use existing package.json
+# Add new options for assets
+begin
+  package_json = Webpacker::PackageJson.config
+  package_json[:scripts][:postinstall] = "npm rebuild node-sass"
+  package_json[:webpacker][:assets] = true
+rescue NoMethodError
+  puts "Error: Webpacker core config and scripts key not found in package.json." \
+       " Make sure webpacker:install is run successfully"
+  exit!
+end
+
+# Write to package.json
+File.open(Rails.root.join("package.json"), "w+") do |file|
+  file.write(JSON.pretty_generate(package_json))
+end
+
+copy_file "#{File.expand_path("..", __dir__)}/config/assets.js",
+          "config/webpack/assets.js"
+
+assets_webpack_config = <<-EOS
+const { webpacker } = require('../../package.json')
+
+if (webpacker.assets) {
+  const assetsConfig = require('./assets.js')
+  sharedConfig.config = merge(sharedConfig.config, assetsConfig)
+}
+EOS
+
+insert_into_file "config/webpack/development.js",
+                  assets_webpack_config,
+                  after: "const sharedConfig = require('./shared.js')\n"
```

Comment:
Don't like this indention style. If that's a Rubocop setting, let's change it. Should look like this:

```ruby
insert_into_file "config/webpack/development.js", 
  assets_webpack_config,
  after: "const sharedConfig = require('./shared.js')\n"
```